### ISSUE TITLE
Include coverage report in SonarCloud analysis

### ISSUE THREAD

**Issue by RUrlus** [RUrlus/carma#58]
Current setup of Sonar analysis does not include the code coverage, it would be nice to have it included in the Sonar analysis as well.

**Comment by RUrlus** [comment]
blocked by #74

**Comment by RUrlus** [comment]
SonorCloud doesn't support actions at time of writing. As Travis has been removed this can be closed
